fecgov/fec-cms
# Add tv ad-example styles to SCSS

## Summary (required)
- Remove CSS from any Wagtail html blocks using the TV ad example iframe pattern
- Add or edit classnames where necessary in html structure  in Wagtail 
- Add to `_responsive-object.scss` and include example html structure there

Resolves #3373

**After deployment:**
Make changes to these three live pages to reflect the simplified structure (right under 'How to test' below). Save as draft to be published after PR is deployed to production as part of a release(existing pages wont break upon deploying this, we just need to change them after):

- [x] Make `after release` ticket to ensure the below items are completed after the  release. **See https://github.com/fecgov/fec-cms/issues/3840**

- [ ] **Edit:** https://fec-prod-proxy.app.cloud.gov/admin/pages/10811/edit/
  **Live page:** https://www.fec.gov/help-candidates-and-committees/making-disbursements-political-party/party-committee-independent-expenditure-tv-disclaimer-ad/

- [ ]  **Edit:** https://fec-prod-proxy.app.cloud.gov/admin/pages/10616//edit/
  **Live page:** https://www.fec.gov/help-candidates-and-committees/making-disbursements/advertising/candidate-television-ad-example/

 - [x] There is a new page  being created by @djgarr,  but will likely not be published before  release and he has been updated on the new structure for making these items in html blocks.

 

## Impacted areas of the application
modified:   fec/static/scss/components/_responsive-object.scss
Live Wagtail pages using the TV ad example pattern. (listed above)


## Screenshots

## How to test
-  checkout and run: `feature/css-styles-tv-examples`
- `npm run-build sass`
- Edit http://127.0.0.1:8000/admin/pages/10811/edit/
- Replace the html block with just this:
```
<div class="video-frame">
   <div class="iframe-container">
    <iframe src="https://www.youtube.com/embed/REOm_O2VJcc" rel=0 frameborder="0" allowfullscreen></iframe>
   </div>
</div>
```
- Preview or publish and confirm that it still renders just like the live version:
https://www.fec.gov/help-candidates-and-committees/making-disbursements/advertising/candidate-television-ad-example/
- Test responsivity  of the patterm

NOTE: If you get error on previewing on your local dev servr, you may need to change `Wagtail >Settings > Sites > Site > Hostname` to localhost. Or you can just publish locally and test that way.
____
